src/main/java/ParkingGarage/Fee.java

```diff
@@ -0,0 +1,78 @@
+package ParkingGarage;
+
+import org.json.JSONObject;
+
+public class Fee extends DataLoaderable {
+	private int id; // fee id
+	private FeeType type; // type of fee (FeeType)
+	private int cost; // cost of the fee
+	
+	// public constructor
+	public Fee(FeeType type, int cost) {
+		if (cost < 0) {
+			throw new IllegalArgumentException("Cost cannot be negative.");
+		}
+		this.type = type;
+		this.cost = cost;
+	}
+	
+	// private constructor
+	private Fee(int id, FeeType type, int cost) {
+		this.id = id;
+		this.type = type;
+		this.cost = cost;
+	}
+	
+	
+	// Getters
+	public int getId() {
+		return id;
+	}
+	
+	public FeeType getType() {
+		return type;
+	}
+	
+	public int getCost() {
+		return cost;
+	}
+	
+	// Static method to load a fee by ID
+	public static Fee load(int id) {
+		try {
+			DataLoader dataLoader = new DataLoader();
+			JSONObject feeJson = dataLoader.getJSONObject("Fee_" + id);
```

Comment:
I think we should keep the top level clean, so all of the `Fee` objects will exist below the `Fee` parent.

For example:
`{
  "fee" : {
    "1": { ... },
    "2": { ... }.
    ...
  }
}`
vs
`{
  "fee_1" : { ... },
  "fee_2": { ... }.
  ...
}`

```suggestion
			JSONObject feeJson = dataLoader.getJSONObject("fee").getJSONObject(id);
```